dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/cloudera/manager/ClouderaCMFHostsTask.java

```diff
@@ -82,19 +77,20 @@ protected Void doRun(
         JsonNode hostsJson;
         try (CloseableHttpResponse hostsResponse =
             httpClient.execute(new HttpGet(hostPerClusterUrl))) {
-          hostsJson = objectMapper.readTree(hostsResponse.getEntity().getContent());
+          hostsJson = getObjectMapper().readTree(hostsResponse.getEntity().getContent());
         }
         String stringifiedHosts = hostsJson.toString();
         writer.write(stringifiedHosts);
         writer.write('\n');
 
-        CMFHostListDTO apiHosts = objectMapper.readValue(stringifiedHosts, CMFHostListDTO.class);
+        CMFHostListDTO apiHosts =
+            getObjectMapper().readValue(stringifiedHosts, CMFHostListDTO.class);
```

Comment:
the same comment about `getObjectMapper` as above

Reply:
ack